packages/aws-cdk-lib/aws-ecs/lib/log-drivers/aws-log-driver.ts

```diff
@@ -116,6 +131,11 @@ export class AwsLogDriver extends LogDriver {
       retention: this.props.logRetention || Infinity,
     });
 
+    var maxBufferSize = undefined;
```

Comment:
We really shouldn't use var here. Honestly we can probably just simplify this whole statement with a ternary operator:

```ts
const maxBufferSize = this.props.maxBufferSize
  ? `${this.props.maxBufferSize.toBytes({ rounding: SizeRoundingBehavior.FLOOR })}b`
  : undefined;
```